tests/test_commands.py

```diff
@@ -169,6 +180,12 @@ def test_drop_existing_db(mock_rethink_db_drop):
     run_drop(args)
 
 
+def test_drop_non_existing_db(mock_rethink_db_drop_not_exist):
+    from bigchaindb.commands.bigchain import run_drop
+    args = Namespace(config=None, yes=True)
+    run_drop(args)
```

Comment:
@vrde I was looking for return values which could be propagated from run_drop() for example, but there are none.
One approach would be to propagate the underlying rethink db status through run_drop (perhaps translating it to some bigchaindb format) and then to the testing code. This could then easily be asserted (looking at the returned `tables_dropped` field, for example).
Regardless, refactoring the wrapper layer at bigchaindb/db/utils.py into a wrapper class could allow a native bigchaindb database status which could be more easily observed by tests (and other api users).
What kind of improvements did you have in mind?